app/components/ui/tale-tab-metadata/template.hbs

```diff
@@ -0,0 +1,91 @@
+<div class="ui form">
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Title</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.title placeholder="Enter a title..." readonly=cannotEditTale}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Authors</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.authors placeholder="Enter the author's name..." readonly="true"}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Category</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.category placeholder="Choose a category..."  readonly=cannotEditTale}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Environment</label>
+    <div class="field thirteen wide column">
+      <div class='ui icon selection dropdown {{if (eq environments.length 0) "loading"}}' style='width:100%'>
+        <input name='TaleImageId' type='hidden' onchange={{action "setTaleEnvironment" value="target.value"}}>
+        <i class='dropdown icon'></i>
+        <div class='default text'> Choose an environment...</div>
+        <div class='menu'>
+          {{#each environments as |env|}}
+            <div class='item' data-value={{env._id}}>
+              <img class='ui image' src={{env.icon}} style='height:28px;width:28px;'> {{env.name}}
+            </div>
+          {{/each}}
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Date Created</label>
+    <div class="field thirteen wide column">
+        {{input value=model.tale.created readonly="true"}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Last Updated</label>
+    <div class="field thirteen wide column">
+      {{input value=model.tale.updated readonly="true"}}
+    </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+      <label class="two wide column right aligned">Description</label>
+      <div class="thirteen wide column">
+        {{ui/markdown-editor markdownValue=model.tale.description}}
+      </div>
+  </div>
+  
+  <div class="inline fields ui grid">
+    <label class="two wide column right aligned">Illustration</label>
+    <div class="ui action field thirteen wide column">
+        {{input type="text" id="tale-icon" placeholder="Tale Icon" value=model.tale.illustration readonly=cannotEditTale }}
+        <div class="ui buttons">
+            <style type="text/css">
```

Comment:
I know we do inline styling 226 other places but I believe we're trying to consolidate it to the files in the styles directory. Small things like display, font-size, etc should be fine inline. @cyberdante correct me if I'm wrong though.

Reply:
This was lazily copied over from the original here: https://github.com/whole-tale/dashboard/blob/master/app/templates/forms/tale-edit.hbs#L39-L44

In my experience with other component-based architectures has been to put the per-component styling in the same directory as the component itself... If we would rather abstract it to a `.css` file elsewhere, that can be arranged as well

Reply:
I'm definitely guilty of not following this pattern, but at the bottom of our README we mention `ember-cli-component-pod` which we should probably be using. Since it's not new code I'm fine with leaving it as-is.

Reply:
@bodom0015 @ThomasThelen: Ember did not provide a specific way of combining in a per-component basis folder every `<component_name>.hbs`, `<component_name>.(s)css`, and `<component_name>.js`, which I also favor. Its original folder structure follows the one by Rails, which was an inspiration for Ember.
Nonetheless, [pods](https://www.npmjs.com/package/ember-cli-component-pod) attempted to solve this situation shortly thereafter. I have personally used this addon [ember-component-css](https://github.com/ebryn/ember-component-css) for a similar contained structure to that previously mentioned, but instead, having:
```
    - app
        - components
            - <component_name>
                component.js
                template.hbs
                styles.scss
```
It is not ideal, but it provides an improvement. It needs some tweaks to the `.scss` files, mainly wrapping the whole file with a certain class (I usually mimic the component's name in `kebab case` and add a `classNames: ['<component_name>']` to the `component.js` file).

Finally, the community was divided with regard to using PODS. Mainly because this structure cannot be ported to other projects seamlessly, but also because it does not work well with addons. Hence, the appearance of _module unification_ and the adoption of _BEM_ for the styles. Sadly, this project had already started when I joined it, and the effort we can devote is driven by the budget that Notre Dame has assigned for my team to work on it. At this point, modifying the whole UI would require a major iteration for which we don't have enough funding, in my opinion.